src/main/kotlin/frc/team5499/frc2018Kotlin/Robot.kt

```diff
@@ -2,25 +2,54 @@ package frc.team5499.frc2018Kotlin
 
 import edu.wpi.first.wpilibj.TimedRobot
 
+import frc.team5499.frc2018Kotlin.controllers.TeleopController
+import frc.team5499.frc2018Kotlin.controllers.AutoController
+import frc.team5499.frc2018Kotlin.subsystems.Drivetrain
+import frc.team5499.frc2018Kotlin.subsystems.Arm
+import frc.team5499.frc2018Kotlin.subsystems.Intake
+
 class Robot : TimedRobot() {
 
     override fun robotInit() {
-        println("Hello World from Kotlin!")
+        super.setPeriod(Constants.UPDATE_PERIOD)
     }
 
-    override fun robotPeriodic() {}
+    override fun robotPeriodic() {
+        Drivetrain.broadcastToDashboard()
```

Comment:
Get rid of this. We will add this functionality later, but probably not using broadcast functions like this.